backend/package.json

```diff
@@ -19,14 +19,13 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "tsc && lint-staged"
+      "pre-commit": "tsc && lint-staged && yarn test"
```

Comment:
Would `npm test` be a better idea to ensure everyone can run it?

Reply:
Fixed!